runtime/zeitgeist/src/parameters.rs

```diff
@@ -312,6 +312,10 @@ parameter_types! {
     // Orderbook parameters
     pub const OrderbookPalletId: PalletId = ORDERBOOK_PALLET_ID;
 
+    // Parimutuel parameters
+    pub const MinBetSize: Balance = 50 * CENT;
```

Comment:
Are there any benefits of setting `MinBetSize` higher than `ExistentialDeposit`?

```suggestion
    pub const MinBetSize: Balance = ExistentialDeposit::get();
```

Reply:
See https://github.com/zeitgeistpm/zeitgeist/pull/1138#discussion_r1370831749